apis/telemetry/v1alpha1/telemetry_types.go

```diff
@@ -68,4 +68,7 @@ type McacTelemetrySpec struct {
 	// Setting it to an empty list will result in all metrics being extracted.
 	// +optional
 	MetricFilters *[]string `json:"metricFilters,omitempty"`
+
+	// enabled sets whether MCAC (legacy metrics endpoint) is enabled.
```

Comment:
We could add a comment stating that MCAC is considered enabled by default, since it might be surprising given that MCAC is legacy.

Reply:
ok, fixed 👍